src/PowerShell/Microsoft.WinGet.Client.Engine/Helpers/OperationWithProgressBase.cs

```diff
@@ -19,6 +20,24 @@ namespace Microsoft.WinGet.Client.Engine.Helpers
     /// <typeparam name="TProgressData">Progress data.</typeparam>
     internal abstract class OperationWithProgressBase<TOperationResult, TProgressData>
     {
+        private static bool isProgressEnabled;
+
+        static OperationWithProgressBase()
+        {
+            // There's an OS bug where COM has never worked properly on arm64, this cause the

```

Comment:
This is a bit broad on the issue, it is an "MBM for structs over a certain size on ARM64" bug.